src/components/organisms/NavBarSchedule/ics.js

```diff
@@ -0,0 +1,290 @@
+/* global saveAs, BlobBuilder */
+/* exported ics */
```

Comment:
Based on the above, it looks like we probably want to use

- https://bundlephobia.com/result?p=ics@2.27.0
  - https://www.npmjs.com/package/ics
  - https://github.com/adamgibbons/ics

`npm install ics`

Reply:
Actually.. somehow I missed this one in my exploration above, which also seems like it might be a decent choice:

- https://bundlephobia.com/result?p=ical-generator@2.1.0 `8.9kB`
  - https://github.com/sebbo2002/ical-generator
    - last updated 7 days ago

Reply:
Awesome, sounds good :)